k8s/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: "Mondoo Kubernetes Cluster Action"
+description: "Scan Kubernetes clusters for vulnerabilities and misconfigurations with Mondoo"
+branding:
+  icon: 'shield'
+  color: 'purple'
+inputs:
+  args:
+    description: >-
+      Additional arguments to pass to Mondoo Client.
+    required: false
+  log-level:
+    description: >-
+      Sets the log level: error, warn, info, debug, trace (default "info")
+    default: info
+    required: false
+  output:
+    description: >-
+      Set the output format for scan results: compact, yaml, json, junit, csv, summary, full, report (default "compact")
+    default: compact
+    required: false
+  score-threshold:
+    description: >-
+      Sets the score threshold for scans. Scores that fall below the threshold will exit 1. (default "0" - job continues regardless of the score returned by a scan).
+    default: '0'
+    required: false
+  service-account-credentials:
+    description: "Base64 encoded service account credentials used to authenticate with Mondoo Platform"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Install Mondoo Client
+      shell: bash
+      run: |
+          echo Installing Mondoo Client...
+          echo ${{ inputs.service-account-credentials }} | base64 -d > mondoo.json
+          curl -sSL https://mondoo.com/install.sh | bash
+    - name: Mondoo status
+      shell: bash
+      run: mondoo status --config mondoo.json
+    - name: Run mondoo scan ${{ inputs.asset-type }}
+      shell: bash
+      run: >
+        mondoo scan k8s 
+        --detect-cicd
```

Comment:
the `--detect-cicd` flag is not needed anymore. I removed it [here](https://github.com/mondoohq/actions/pull/17#discussion_r948877697)

Reply:
that applies for all scan types

Reply:
Thank you for letting me know. i will remove `detect-cicd` from all scan jobs

Reply:
👍